eLibrary.ru.js

```diff
@@ -291,7 +293,7 @@ var testCases = [
 				],
 				"date": "2012",
 				"ISSN": "2219-5335",
-				"issue": "13",
+				"issue": "13 (1)",
```

Comment:
Is that volume and issue? Then we should try to split this up into two fields, whenever we see such a pattern.

However, this could also simply be the "issue number" relative to this year, i.e. it is overall the issue 13 but in the year 2012 it is issue 1.

Reply:
[This](https://elibrary.ru/contents.asp?id=33729006&selid=17339044) looks like this is the "issue number relative to this year":
![image](https://user-images.githubusercontent.com/920364/60020277-64b09400-9698-11e9-8a35-0b5fd43deef9.png)

However... [Here](https://elibrary.ru/contents.asp?titleid=10023)'s another journal, which uses similar format, but now "issue number relative to this year" comes first:
![image](https://user-images.githubusercontent.com/920364/60020387-99245000-9698-11e9-8b31-6b37f23290c9.png)

Thus, I'm not sure if it's safe to make any assumptions about the contents of that field.

Reply:
Okay, then we can leave it as it comes from the website, which then maybe is be the best we can do in a general approach.